testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/ejb/remote/RemoteFailoverTestCase.java

```diff
@@ -294,7 +299,10 @@ public void testConcurrentFailover() throws Exception {
                 Future<?> future = executor.scheduleWithFixedDelay(new IncrementTask(bean, count, latch), 0, INVOCATION_WAIT, TimeUnit.MILLISECONDS);
                 latch.await();
 
-                undeploy(this.findDeployment(target));
+                String deployment = this.findDeployment(target);
+                System.out.println("--- Undeploying: " + deployment + " ---");
+                undeploy(deployment);
+                // undeploy(this.findDeployment(target));
```

Comment:
Are these intentional? If anything you could put the loggin in the org.jboss.as.test.clustering.cluster.ClusterAbstractTestCase#undeploy method.